src/Tooling/Bit.Tooling.Templates/TodoTemplate/Docs/Introduction.md

```diff
@@ -0,0 +1,37 @@
+# Introduction
```

Comment:
These contents are redundant. We'll have contents like these in readme.md in the repository's root folder and in bitplatform.dev site

Reply:
Entirely removed